Update common-sso-openidconnect.rst (#5638)
mattermost/docs: source/onboard/common-sso-openidconnect.rst

8. Restart your Mattermost server to see the changes take effect.

.. note::
  When Mattermost is configured to use OpenID Connect for user authentication, the following user attribute changes can't be made through the Mattermost API: first name, last name, or username. OpenID Connect must be the authoritative source for these user attributes.
  

>   Frequently Asked Questions

Reviewer: ```suggestion
Frequently Asked Questions
```

--------------------------

How can I use LDAP attributes or Groups with OpenID?
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

> At this time, LDAP data is not compatible with OpenID. If you currently rely on LDAP to manage your users' teams, channels, groups, or attributes, you will not be able to do this automatically with users who have logged in with OpenID. If you need LDAP synced to each user, we suggest using SAML or LDAP as the login provider. Some OpenID providers can use SAML instead, like Keycloak.

Reviewer: ```suggestion
At this time, LDAP data isn't compatible with OpenID. If you currently rely on LDAP to manage your users' teams, channels, groups, or attributes, you won't be able to do this automatically with users who have logged in with OpenID. If you need LDAP synced to each user, we suggest using SAML or LDAP as the login provider. Some OpenID providers can use SAML instead, like Keycloak.
```
